GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/contentpage.config

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Each document type to demonstrate a component is named after that component, so 'Content page' is the wrong name here; it should be 'Breadcrumbs'. We do this so that when we need to demonstrate code interacting with the component we can add a controller specifically for that page.

Reply:
Sorted! Although I would like to discuss why not having a multi component page to try out variation of components and their impact on the block grid.